perun-core/src/main/java/cz/metacentrum/perun/core/impl/Utils.java

```diff
@@ -1089,7 +1089,7 @@ public static void sendSMS(PerunSession sess, Member member, String message) thr
 	 * @throws IllegalArgumentException when the phone or message has a wrong format
 	 */
 	public static void sendSMS(String telNumber, String message) throws InternalErrorException {
-		log.info("Sending SMS with text \"" + message + "\" to tel. number " + telNumber + ".");
+		log.debug("Sending SMS with text \"" + message + "\" to tel. number " + telNumber + ".");
```

Comment:
Please pass ``message`` and ``telNumber`` like params into debug method and not as joined string with ``+``.